pkg/systemd/services.css

```diff
@@ -0,0 +1,18 @@
+.comma-list {
+  display: inline;
+  list-style: none;
+  padding-left: 0;
+}
+
+.comma-list > li
+{
+  display: inline;
+}
+
+.comma-list > li:after {
+  content: ",";
+}
+
+.comma-list > li:last-child:after {
+  content: "";
+}
```

Comment:
Combine these two CSS rules (the `.comma list > li` ones) into something like:

```css
.comma-list > li:not(:last-child):after {
  content: ",";
}
```

FWIW: I think it should be possible to also write `.comma-list > li:after:not(:last-child)`, but I think the above order makes a bit more sense.